src/lib/permission-exclusions-keto-tuples.ts

```diff
@@ -28,40 +28,32 @@
  ******/
 
 import * as keto from '@ory/keto-client'
-import { PatchDelta, InternalRelationTuple } from '@ory/keto-client'
 import Config from '../shared/config'
 import { logger } from '../shared/logger'
+import { KETO_NAMESPACES, KETO_RELATIONS, PAGE_SIZE } from '../constants'
 
 class KetoTuples {
-  oryKetoReadApi: keto.ReadApi;
-  oryKetoWriteApi: keto.WriteApi;
+  relationshipApi: keto.RelationshipApi
+  adminRelationshipApi: keto.RelationshipApi
 
   constructor () {
-    this.oryKetoReadApi = new keto.ReadApi(
-      undefined,
-      Config.ORY_KETO_READ_SERVICE_URL
-    )
-    this.oryKetoWriteApi = new keto.WriteApi(
-      undefined,
-      Config.ORY_KETO_WRITE_SERVICE_URL
-    )
+    this.relationshipApi = new keto.RelationshipApi(undefined, Config.ORY_KETO_READ_SERVICE_URL)
+    this.adminRelationshipApi = new keto.RelationshipApi(undefined, Config.ORY_KETO_WRITE_SERVICE_URL)
   }
 
   /**
    * Gets all permission exclusions in the namespace from Ory Keto server.
    */
   async getAllPermissionExclusionCombos () : Promise<string[]> {
-    const response = await this.oryKetoReadApi.getRelationTuples(
-      'permission',
-      undefined,
-      'excludes',
-      undefined,
-      undefined,
-      1000000
-    )
+    const response = await this.relationshipApi.getRelationships({
+      namespace: KETO_NAMESPACES.permission,
+      relation: KETO_RELATIONS.excludes,
+      pageSize: PAGE_SIZE
+    })
+
     const relationTuples = response.data?.relation_tuples || []
     return relationTuples.map(tuple => {
-      const permissionA = tuple.subject.replace(/permission:([^#.]*)(#.*)?/, '$1')
+      const permissionA = tuple.subject_id?.replace(/permission:([^#.]*)(#.*)?/, '$1')
```

Comment:
I think this will be subject_set

Reply:
what benefits will we have from usage of `subject_set` instead of `subject_id`?